setup.py

```diff
@@ -344,7 +344,7 @@ def run(self):
     NUMPY_STR,
     'scipy >= 1.7.0',
     'smart_open >= 1.8.1',
-    'FuzzyTM >= 0.4.0'
+
```

Comment:
No need for the blank line.

Reply:
Haha, thanks for your help and patience @piskvorky! I have replied to [your comment in the original thread](https://github.com/RaRe-Technologies/gensim/pull/3424#issuecomment-1377337579).